tests/unit/models/reddit/test_removal_reasons.py

```diff
@@ -30,7 +29,10 @@ def test_equality(self):
         assert reason1 != reason2
         assert reason1 != reason3
         assert reason1 == reason4
-        assert reason1 != reason5
+
+    def test_exception(self):
+        with pytest.raises(ValueError):
+            RemovalReason(self.reddit, subreddit=self.reddit.subreddit("a"))
```

Comment:
You might as well also test the case where both `id` and `_data` are given (in a second `pytest.raises` block).